hvpy/api_groups/jpeg2000/tests/test_get_jp2_image.py

```diff
@@ -6,68 +6,56 @@
 from hvpy.api_groups.jpeg2000.get_jp2_image import getJP2ImageInputParameters
 from hvpy.core import execute_api_call
 
-
-def test_str_response():
-    date_obj = datetime(2022, 1, 1, 23, 59, 59)
-    params = {"date": date_obj, "sourceId": 14, "jpip": True, "json": False}
-    params = getJP2ImageInputParameters(**params)
-    response = execute_api_call(input_parameters=params)
-    assert isinstance(response, str)
-    assert response.startswith("jpip://")
-
-
-def test_json_response():
-    date_obj = datetime(2022, 1, 1, 23, 59, 59)
-    params = {"date": date_obj, "sourceId": 14, "jpip": True, "json": True}
-    params = getJP2ImageInputParameters(**params)
-    response = execute_api_call(input_parameters=params)
-    assert isinstance(response, dict)
-    assert "uri" in response
-    assert response["uri"].startswith("jpip://")
+date_obj = datetime(2020, 1, 1)
```

Comment:
This isn't what I had in mind when I said parametrize the tests. Can you undo all of this?

Reply:
I don't think this makes the tests better.

Reply:
Ok